types/artifacts.d.ts

```diff
@@ -489,14 +489,17 @@ declare global {
         boundingRect: Rect;
       }
 
+      export type TraceEventType = 'largest-contentful-paint'|'cumulative-layout-shift'|'animation';
```

Comment:
let's just do this inline so we don't have a `LH.Artifacts.TraceEventType` floating around when it's specific to our trace elements gatherer :)